rpcserver.go

```diff
@@ -2705,6 +2733,33 @@ func (r *rpcServer) AddFederationServer(ctx context.Context,
 
 	serversToAdd := fn.Map(in.Servers, unmarshalUniverseServer)
 
+	// Before we add the server as a federation member, we check that we can
+	// actually connect to it and that it isn't ourselves.
+	for idx := range serversToAdd {
```

Comment:
I think this loop + info check prevents fixes the issue when trying to add ourself via RPC, but not via config file / at startup. I think this should be moved deeper into the universe code, into `AddServer`:

https://github.com/lightninglabs/taproot-assets/blob/e74932ba9182a773f68252fe6a0f4b34144443c9/universe/auto_syncer.go#L364

So that it's also executed at start.

Reply:
Hmm, I somehow managed to completely forget that we had that other code path from the config side. Which was the whole point of https://github.com/lightninglabs/taproot-assets/issues/350 now that I think about it... Anyway, changed things a bit now to include that other code path as well.